identity-verification/module_2/2.1 Rekognition IDV API Examples.ipynb

```diff
@@ -0,0 +1,736 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Key Identity Verification APIs\n",
+    "\n",
+    "-------\n",
+    "\n",
+    "This lab provides hands on experience with the key Amazon Rekognition APIs for identity verification. \n",
+    "\n",
+    "## Introduction \n",
+    "\n",
+    "-------\n",
+    "\n",
+    "In-person user identity verification is slow to scale, costly, and high friction for users. Machine learning powered facial biometrics can enable online user identity verification. Amazon Rekognition offers pre-trained facial recognition and analysis capabilities that you can quickly add to your user onboarding and authentication work flows to verify opted-in users' identity online. \n",
+    "\n",
+    "In this notebook, we'll use the Amazon Rekgonition's key APIs for Identity Verification. After running this notebook you should be able to use the following APIs:\n",
+    "\n",
+    "- DetectFaces: DetectFaces detects the 100 largest faces in the image. \n",
+    "- CompareFaces: Compares a face in the source input image with each of the 100 largest faces detected in the target input image.\n",
+    "- CreateCollection: Creates a searchable index of faces. \n",
+    "- IndexFaces: Detects faces in the input image and adds them to the specified collection and returns a face id which can be used in subsequent searches. \n",
+    "- SearchFacesByImage: For a given input image, first detects the largest face in the image, and then searches the specified collection for matching faces.\n",
+    "- SearchFaces: For a given input face ID, searches for matching faces in the collection of the face it belongs to.\n",
+    "- DeleteFaces: Deletes faces from a collection. You specify a collection ID and an array of face IDs to remove from the collection.\n",
+    "\n",
+    "\n",
+    "###### Sample Images from \"Labeled Faces in the Wild\":\n",
+    "-----\n",
+    "*Gary B. Huang, Manu Ramesh, Tamara Berg, and Erik Learned-Miller.\n",
+    "Labeled Faces in the Wild: A Database for Studying Face Recognition in Unconstrained Environments.\n",
+    "University of Massachusetts, Amherst, Technical Report 07-49, October, 2007* [pdf](\"http://vis-www.cs.umass.edu/lfw/lfw.pdf\")\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import io\n",
+    "import boto3\n",
+    "import json\n",
+    "from IPython.display import HTML, display, Image as IImage\n",
+    "from PIL import Image, ImageDraw, ExifTags, ImageColor, ImageFont\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
```

Comment:
These imports aren't used anywhere. Can they be removed?

Reply:
done